Mathlib/Order/LiminfLimsup.lean

```diff
@@ -281,6 +298,63 @@ theorem isCobounded_principal (s : Set α) :
 theorem IsCobounded.mono (h : f ≤ g) : f.IsCobounded r → g.IsCobounded r
   | ⟨b, hb⟩ => ⟨b, fun a ha => hb a (h ha)⟩
 
+/-- For nontrivial filters in linear orders, coboundedness for `≤` implies frequent boundedness
+from below. -/
+lemma IsCobounded.frequently_ge [LinearOrder α] [NeBot f] (cobdd : IsCobounded (· ≤ ·) f) :
+    ∃ l, ∃ᶠ x in f, l ≤ x := by
+  obtain ⟨t, ht⟩ := cobdd
+  by_cases tbot : IsBot t
+  · refine ⟨t, Frequently.of_forall fun r ↦ tbot r⟩
+  obtain ⟨t', ht'⟩ : ∃ t', t' < t := by
+    by_contra!
+    exact tbot this
```

Comment:
This is a common case split, so we have a dedicated lemma for it
```suggestion
  obtain tbot | ⟨t', ht'⟩ := isBot_or_exists_lt t
  · exact ⟨t, .of_forall fun r ↦ tbot r⟩
```
Same below

Reply:
Nice trick. I did some minor additional golfing in the same area (`isBoundedUnder_le_sum`, `isBoundedUnder_ge_sum`).